contract-api-reference/publicresolver.md

```diff
@@ -64,6 +65,44 @@ Emits the following event:
 event AddrChanged(bytes32 indexed node, address a);
 ```
 
+## Get Blockchain Address
+
+```text
+function addr(bytes32 node, uint coinType) external view returns(bytes memory);
+```
+
+Returns the Blockchain address associated with the provided `node` and `cointType`, or 0 if none.
+
+This function has interface ID _0xf1cb7e06_.
+
+This function is specified in [EIP 137](https://eips.ethereum.org/EIPS/eip-137).
+
+To convert the binary representation into the address, use `formatsByCoinType[0].encoder(binary)` of  [address-encoder](https://github.com/ensdomains/address-encoder)
```

Comment:
A slightly more verbose description of the format here would be good - you could lift it from the EIP.

Reply:
Done